index.js

```diff
@@ -386,7 +386,7 @@
         return x != null && (
           R.type(type.prototype['@@type']) === 'String' ?
             x['@@type'] === type.prototype['@@type'] :
-            R.type(x) === R.nth(1, R.match(/^function (\w*)/, String(type)))
+            R.type(x) === R.nth(1, R.match(/function (\w*)/, String(type)))
```

Comment:
Actually, you can `git commit --amend` and tweak the commit message. Then `git push origin <branch> --force` to update this pull request. You can then update the pull request's title in the GitHub UI.